qa/L0_metrics/test.sh

```diff
@@ -132,12 +131,42 @@ fi
 kill_server
 set -e
 
+### General metrics tests
+
+set +e
+CLIENT_PY="./general_metrics_test.py"
+CLIENT_LOG="general_metrics_test_client.log"
+SERVER_LOG="general_metrics_test_server.log"
+SERVER_ARGS="$BASE_SERVER_ARGS --log-verbose=1"
+PYTHON_TEST="general_metrics_test.py"
+run_and_check_server
+# Test 1 for normal mode
+python3 ${PYTHON_TEST} TestGeneralMetrics.test_metrics_load_time -v 2>&1 | tee ${CLIENT_LOG}
+kill_server
+
+set +e
+CLIENT_PY="./general_metrics_test.py"
+CLIENT_LOG="general_metrics_test_client.log"
+SERVER_LOG="general_metrics_test_server.log"
+SERVER_ARGS="$BASE_SERVER_ARGS --model-control-mode=explicit --log-verbose=1"
+run_and_check_server
+export MODEL_NAME='libtorch_float32_float32_float32'
+code=`curl -s -w %{http_code} -X POST ${TRITONSERVER_IPADDR}:8000/v2/repository/models/${MODEL_NAME}/load`
+# Test 2 for explicit mode LOAD
+python3 ${PYTHON_TEST} TestGeneralMetrics.test_metrics_load_time_explicit_load -v 2>&1 | tee ${CLIENT_LOG}
```

Comment:
Can you use the pytest style `python3 -m pytest --junitxml ...` so we get the unit test report on gitlab? Will need to tweak your class to inherit from `unittest.TestCase` directly instead of our wrapper `tu.TestResultCollector`, see https://github.com/triton-inference-server/server/blob/main/qa/L0_metrics/pinned_memory_metrics_test.py for a reference